wm/frame.go

```diff
@@ -342,21 +338,33 @@ func (f *frame) decorate(force bool) {
 		0, 0, int16(f.width-iconAndBorderSizePix), 0, uint16(iconAndBorderSizePix), uint16(iconSizePix))
 }
 
-func (f *frame) applyTheme(force bool) {
-	if !f.framed {
+func (f *frame) decorateBorderless() {
```

Comment:
This new method confuses me. Is it for borderless windows? if so isn't that the same as not decorated? so the chunk below is not executed?
After that it does a resize but only if it's fullscreen.
Can you help me understand? (possibly update the name to reflect what it does?)

Reply:
Yes it is for borderless windows.  I changed the name of the method to applyBorderlessTheme
This will applies the theme for borders/decorations if _MOTIF_WM_HINTS says so - or it will apply a completely borderless frame and theme (basically just a rectangle the same size as the window that is theme bg colored)

Fullscreen requires the additional resize, as was the case there in applyTheme() before this PR, but the additional resize is not necessary/will not render borderless correctly if not fullscreen and not decorated.